x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlDailyMaintenanceService.java

```diff
@@ -154,7 +187,7 @@ private static TimeValue delayToNextTime(ClusterName clusterName) {
     }
 
     public synchronized void start() {
-        logger.debug("Starting ML daily maintenance service");
+        logger.info("Starting ML daily maintenance service");
```

Comment:
```suggestion
        logger.debug("Starting ML daily maintenance service");
```

For consistency with the logging for the `stop()` function.